site/content/docs/5.2/components/modal.md

```diff
@@ -244,6 +244,13 @@ When backdrop is set to static, the modal will not close when clicking outside o
 
 When modals become too long for the user's viewport or device, they scroll independent of the page itself. Try the demo below to see what we mean.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
```

Comment:
The idea is to avoid as much as possible these very long modals. It's a matter of conception. Now, if this can't be avoid, I guess this feature would be used anyway...

Reply:
Decided: we let this callout message + replace "can not" by "should not"

Reply:
Replace "These variants" by "This first variant"